pkg/client/client_test.go

```diff
@@ -344,7 +350,11 @@ func MustFileInTempDir(parent, name string) string {
 	return file
 }
 
-func configGivenEnv(serverRun, runtimeRoot string) *client.ConmonClient {
+func configGivenEnv(serverRun, runtimeRoot string, terminal bool) *client.ConmonClient {
+	logDriver := "systemd"
+	if terminal {
+		logDriver = "stdout"
+	}
```

Comment:
can you explain the motivation here?

Reply:
We cannot log to stdout any more because the logger would not log anything because of the disconnected streams.

Reply:
this is the log driver for conmon-rs logs, not for contianer logs. I would expect each container to be able to disconnect and not affect the outer conmon-rs or other containers. that's the case, right?

Reply:
Yeah it is the case, not sure how to avoid this right now :thinking:

Reply:
we may want an issue to track to make sure nothing is funky here